Makefile

```diff
@@ -33,11 +33,11 @@ getdeps:
 
 verify: getdeps govet gotest lint
 
-operator:
+operator: verify
 	@CGO_ENABLED=0 GOOS=linux go build -trimpath --ldflags $(LDFLAGS) -o minio-operator
 
-docker:
-	@docker build -t $(TAG) .
+docker: operator logsearchapi
```

Comment:
build already does this, `make docker` ideally should only do the container, this is useful for building containers locally

Reply:
The reason is because in CI/CD `make docker` was being used.

Reply:
But binaries should be built as part of dependency so that each Make target is self contained.

Make targets should never have independent builds i.e it is unexpected in make world to do

```
make target1 
make target2

make docker
```

It is only expected to have `install` and `all` separately that's all..